xlators/features/shard/src/shard.c

```diff
@@ -548,19 +548,21 @@ shard_local_wipe(shard_local_t *local)
 }
 
 int
-shard_modify_size_and_block_count(struct iatt *stbuf, dict_t *dict)
+shard_modify_size_and_block_count(struct iatt *stbuf, dict_t *dict,
```

Comment:
nit: if you are already changing the signature of the function, make it static while at it.

Reply:
Done